core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -19,104 +19,103 @@
   <div id="showAdditionalMetrics"></div>
   <h4 class="title-table">Summary</h4>
   <div class="container-fluid">
-    <div class="container-fluid">
-      <table id="summary-execs-table" class="table table-striped compact cell-border">
-        <thead>
-        <th></th>
-        <th>RDD Blocks</th>
-        <th><span data-toggle="tooltip"
-                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory.">Storage Memory</span>
-        </th>
-        <th>
-          <span data-toggle="tooltip"
-                title="Memory used / total available memory for on heap storage of data like RDD partitions cached in memory.">On Heap Storage Memory</span>
-        </th>
-        <th>
-          <span data-toggle="tooltip"
-                title="Memory used / total available memory for off heap storage of data like RDD partitions cached in memory.">Off Heap Storage Memory</span>
-        </th>
-        <th>Disk Used</th>
-        <th>Cores</th>
-        <th><span data-toggle="tooltip" data-placement="top" title="Number of tasks currently executing. Darker shading highlights executors with more active tasks.">Active Tasks</span></th>
-        <th><span data-toggle="tooltip" data-placement="top" title="Number of tasks that have failed on this executor. Darker shading highlights executors with a high proportion of failed tasks.">Failed Tasks</span></th>
-        <th>Complete Tasks</th>
-        <th>Total Tasks</th>
-        <th><span data-toggle="tooltip"
-                  title="Shaded red when garbage collection (GC) time is over 10% of task time">
-          Task Time (GC Time)</span></th>
-        <th><span data-toggle="tooltip"
-                  title="Bytes and records read from Hadoop or from Spark storage.">Input</span></th>
-        <th><span data-toggle="tooltip"
-                  title="Total shuffle bytes and records read (includes both data read locally and data read from remote executors).">
-          Shuffle Read</span></th>
-        <th>
-          <span data-toggle="tooltip" data-placement="top"
-                title="Bytes and records written to disk in order to be read by a shuffle in a future stage.">
-            Shuffle Write</span>
-        </th>
-        <th>
-          <span data-toggle="tooltip" data-placement="top"
-                title="Number of executors blacklisted by the scheduler due to task failures.">
-            Blacklisted</span>
-        </th>
-        </thead>
-        <tbody>
-        </tbody>
-      </table>
-    </div>
-  </div>
-  <h4 class="title-table">Executors</h4>
-  <div class="container-fluid">
-    <div class="container-fluid">
-      <table id="active-executors-table" class="table table-striped compact cell-border">
-        <thead>
+    <table id="summary-execs-table" class="table table-striped compact cell-border" style="width: 100%">
```

Comment:
It seems that this file only removes the element ` <div class="container-fluid">` and add `style="width: 100%"`
Could you keep the other part of the file unchanged so that it is easier for review?

Reply:
Thank you for reviewing the PR.  I just submitted a PR to review the indent formatting updates to make the review easier.